com.cinecred.cinecred.yml

```diff
@@ -0,0 +1,41 @@
+id: com.cinecred.cinecred
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: cinecred
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+modules:
+  - name: ghostscript
+    config-opts:
+      - --disable-cups
+      - --disable-dbus
+      - --disable-gtk
+    cleanup:
+      - /share/doc
+      - /share/man
+    sources:
+      - type: archive
+        url: https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs10040/ghostscript-10.04.0.tar.gz
+        sha256: c764dfbb7b13fc71a7a05c634e014f9bb1fb83b899fe39efc0b6c3522a9998b1
+  - name: cinecred
+    buildsystem: simple
+    build-commands:
+      - sed -i "s/Icon=cinecred/Icon=$FLATPAK_ID/" cinecred.desktop
+      - sed -i "s/>cinecred/>$FLATPAK_ID/" cinecred.metainfo.xml
```

Comment:
if you install it with its original name and put `rename-appdata-file` in the top-level of the manifest, you don't have to do that obscure sed command. Also install the desktop file with its origianal name and use `rename-desktop-file` and it does the right thing too.